pkg/controllers/v1alpha2/mpi_job_controller.go

```diff
@@ -1158,3 +1165,8 @@ func newLauncher(mpiJob *kubeflow.MPIJob, kubectlDeliveryImage string) *batchv1.
 		},
 	}
 }
+
+func setRestartPolicy(podTemplateSpec *corev1.PodTemplateSpec, spec *kubeflow.ReplicaSpec) {
+	podTemplateSpec.Spec.RestartPolicy = corev1.RestartPolicy(spec.RestartPolicy)
+
```

Comment:
Remove redundant line

Reply:
Done. Another will set the default value to cleanpodpolicy through default.go.